esp-hal/src/spi/slave.rs

```diff
@@ -423,32 +407,45 @@ where
         self.enable_dma();
 
         reset_dma_before_load_dma_dscr(reg_block);
-        tx.prepare_transfer_without_start(
```

Comment:
I don't know if this is the diff being weird, but did you mean to remove this block?

Reply:
It was moved called few lines later because of ESP32 - since we are not adding ESP32 I reverted this change.